{
 "bad": [
  "_version.get_versions",
  "errno.EEXIST",
  "google_auth_oauthlib.flow.InstalledAppFlow.from_client_config",
  "json.dump",
  "json.load",
  "json.loads",
  "logging.getLogger",
  "os.environ",
  "os.makedirs",
  "os.name",
  "os.path.dirname",
  "os.path.exists",
  "os.path.expanduser",
  "os.path.join"
 ],
 "deps": {
  "_version": [],
  "errno": [],
  "google_auth_oauthlib": [],
  "json": [],
  "logging": [],
  "os": [],
  "pydata_google_auth": [
   "pydata-google-auth/conda-forge/noarch/pydata-google-auth-0.1.3-py_0"
  ]
 }
}